README.md

```diff
@@ -19,6 +31,8 @@ $ brew install swift-outdated
 
 Since `swift-outdated` installs with its name, it can be called just like a subcommand of Swift itself via `swift outdated`.
 
+If you installed `swift-outdated` via Mint, it has to be called using mint like `mint run outdated`.
```

Comment:
This doesn't appear to be true, it can be run like that, but it appears if you have your mint installation dir (~/.mint/bin) in your PATH, you can just run it like you would expect.

Reply:
Oh, I see this makes sense for people using a Mintfile, but I'd argue they know what they're doing 😅